### PR TITLE
Add support generating Deprecated annotations and docs

### PR DESCRIPTION
**Pull request by altro3**
Removed support swagger1 annotations for java generator

### CODE REVIEW

File: openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ protected String generateFiles(MicronautCodeGenerator<?> codegen, String configP
         } catch (IOException e) {
             fail("Unable to create temporary directory for output");
         }
-        output.deleteOnExit();
+//        output.deleteOnExit();
```
Comment (sdelamo) [inline openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java]:
```suggestion
        output.deleteOnExit();
```

**Review by sdelamo — CHANGES_REQUESTED** [review]
removing swagger 1 annotations is a breaking change. 

Why are we mixing this removal with a PR to add Deprecated annotations?

File: openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java (hunk 1 above)
Comment (altro3) [reply openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java]: fixed

**Comment by altro3** [comment]
> removing swagger 1 annotations is a breaking change.
> 
> Why are we mixing this removal with a PR to add Deprecated annotations?
You are wrong, this change is not breaking. No one could generate code with swagger1 annotations before, simply because no one had the ability to change this setting (it is not passed in the plugin for gradle and maven).

In fact, this is just garbage that was left after moving the generator to this repository and this code was only in templates. By the way, in the generator for Kotlin, which I already wrote myself, there was no support for the first swagger.

I just removed the extra garbage, and this will not affect the previously generated code in any way.

**Review by graemerocher — APPROVED** [review]

**Comment by graemerocher** [comment]
please resolve the conflicts

**Comment by altro3** [comment]
@graemerocher done

**Review by graemerocher — APPROVED** [review]

**Pull request merged**